metadata-server/metadata-server.cabal

```diff
@@ -40,6 +40,7 @@ executable metadata-server
                         -Wincomplete-uni-patterns
                         -Wno-unsafe
                         -threaded
+                        -rtsopts "-with-rtsopts=-T -I0"
```

Comment:
Good to have `-rtsopts` in there - pretty much all executables should have it.

But this needs a comment about why [`-I0`](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/runtime_control.html#rts-flag--I%20%E2%9F%A8seconds%E2%9F%A9) is needed. I suppose it's something to do with the leak?